spec/features/user_changes_password_spec.rb

```diff
@@ -0,0 +1,32 @@
+feature "User changes password" do
+  before :each do
```

Comment:
I prefer not to use `before` blocks, because they tend to introduce the Mystery Guest test smell. Details here: http://robots.thoughtbot.com/lets-not.